app/controllers/evaluations_controller.rb

```diff
@@ -81,6 +88,23 @@ def mark_complete
     end
   end
 
+  def recuse
+    @evaluator_submission_assignment = find_evaluator_submission_assignment
+    return unauthorized_redirect unless can_access_evaluation?
+
+    begin
+      destroy_recused_evaluation
+      if recuse_evaluator
```

Comment:
destroying the record first and then updating the status could lead to deleting an evaluation and the status update failing. this should either be done inside a transaction or check the update was successful first:
```suggestion
      destroy_recused_evaluation
      if recuse_evaluator
```
```suggestion
      if recuse_evaluator
        destroy_recused_evaluation
```